Projects/App/Sources/Journeys/LoggedInNavigation.swift

```diff
@@ -382,13 +383,14 @@ struct HomeTab: View {
                         case let .claimDetail(id):
                             let claimStore: ClaimsStore = globalPresentableStoreContainer.get()
                             if let claim = claimStore.state.claim(for: id) {
-                                ClaimDetailView(claim: claim)
+                                ClaimDetailView(claim: claim, fromChat: true)
                                     .configureTitle(L10n.claimsYourClaim)
+
                             }
                         }
                     }
                 )
-
+                .environmentObject(homeNavigationVm.router)
```

Comment:
If we return 
@StateObject var router = Router()
then this is not necessary. Also I think this homeNavigationVm router is router that we use when we want to push something in the home view. 
It would be best to just keep router in the ChatNavigation as an @StateObject